runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -165,8 +163,6 @@ def commonLegacyExcludeCategories = [
 def commonRunnerV2ExcludeCategories = [
         'org.apache.beam.sdk.testing.UsesCommittedMetrics',
         'org.apache.beam.sdk.testing.UsesGaugeMetrics',
-        'org.apache.beam.sdk.testing.UsesSetState',
```

Comment:
Are `MapState` and `SetState` also supported in dataflow runner v2(unified worker) within this changes?

Reply:
Done